extensions/libraries/redcore/form/fields/rmedia.php

```diff
@@ -63,7 +63,7 @@ protected function getInput()
 			$script = array();
 			$script[] = '	function jInsertFieldValue(value, id) {';
 			$script[] = '		var old_value = document.id(id).value;';
-			$script[] = '		if (old_value != value) {';
+			$script[] = '		if (old_value != value && value) {';
```

Comment:
Sorry about the second one, this is javascript.  But please invert the order to check the value being set in the first place